packages/plugin-core/src/commands/GoToNoteInterface.ts

```diff
@@ -31,4 +32,5 @@ export type GoToNoteCommandOutput =
   | { kind: TargetKind.NOTE; note: NoteProps; pos?: Position; source?: string }
   // When opening a non-note file
   | { kind: TargetKind.NON_NOTE; fullPath: string; type: GotoFileType }
+  | { kind: TargetKind.LINK; fullPath: string }
```

Comment:
Could you add a comment for in what case this kind is supposed to be used, similar to the existing comments for the other kinds?